substrafl/remote/register/register.py

```diff
@@ -37,7 +37,7 @@
 """
 
 _GPU_BASE_IMAGE = """
-FROM nvidia/cuda:11.8.0-runtime-ubuntu22.04
+FROM nvidia/cuda:12.6.1-runtime-ubuntu24.04
```

Comment:
11.8.0 -> 12.6.1 seems like a huge bump - are we sure there was not a reason for using 11.8.0?

Reply:
11.8 is not supported in Ubuntu 24.04.
Having to use Ubuntu 22.04 is forcing us to either install a vulnerable version of `libpixman-1-dev` (a dependency of openslides-tool) or to install `libpixman-1-dev` from source (I must admit, I haven't investigated much further this option).
Current versions of pytorch are still supporting CUDA 11.8, but it's the oldest version supported.

I can try with cuda 12.5.1, which is the oldest CUDA supported by Ubuntu 24.04 https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/supported-tags.md

Reply:
Tested on dev ✅